readme.md

```diff
@@ -101,3 +101,176 @@ In the console...
   - Using `each`, iterate over those songs and for each song, print "I like" + the song name
 
 ## Part 3 - Build a RESTful Interface using Sinatra
+
+First you wrote the schema for a database to store records of artists and their songs for a music catalog app. You then seeded the database with sample data and we were able to interact with that data (performing CRUD operations) using SQL queries in Postgresql. Next, you connected ActiveRecord to the database which enhanced your ability to interact with the data in a Ruby environment. Still, our only way to interact with the data is in a console.
+
+Today, you are going to use Sinatra to listen for HTTP requests and serve HTML (rendered from ERB templates) in response.
+
+### Part 3.1: Root Route, Home View, and Layout View
+
+1. Run your application: `ruby app.rb` and go to localhost:4567 in your browser... Sinatra doesn't know this ditty.
```

Comment:
Students can checkout the lab-starter branch for starter code. The working app.rb is in there with the necessary Gemfile, db/ files and models (everything they would have done if they followed along in class).